content/php/concepts/string-functions/terms/html-entity-decode/html-entity-decode.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: 'PHP html_entity_decode() Function'
+Description: 'Converts HTML entities to characters.'
+Subjects: 
+- 'Web Design'
+- 'Web Development'
+Tags:
+- 'Functions'
+- 'Characters'
+CatalogContent:
+- 'php/concepts/string-functions/terms/html-entity-decode'
+---
+
+
+**'html_entity_decode()'** is the opposite of **'htmlentities()'** in that it converts HTML entities in the string to their corresponding characters.
+
+## Syntax
+```pseudo
+html_entity_decode(str, flags, character-set)
+```
+- 'string - Required. Specifies the string to decode'
+- 'flags - Optional. Specifies how to handle quotes and which document type to use.'
+- c'haracter-set - Optional. A string that specifies which character-set to use.'
+
+## Example
+```php
+<?php
+$orig = "dog";
+$a = htmlentities($orig);
+$b = html_entity_decode($a);
+echo $a;
+echo "\n";
+echo $b;
+?>
+```
+
```

Comment:
Add an output block after this example. Like this:
```shell 
output
```
<img width="106" alt="image" src="https://github.com/Codecademy/docs/assets/94860622/b4f808fa-2284-4899-b7c7-e5138e82ff49">

Reply:
@dakshdeepHERE Added.